Equilobe.DailyReport.SL/DbService.cs

```diff
@@ -55,5 +58,27 @@ public static string GetReportTime(string baseUrl, long projectId)
                 .Select(qr => qr.ReportTime)
                 .FirstOrDefault();
         }
+
+        public static void SetExecutionDate(long id)
```

Comment:
DbService (vom redenumi in DataService) insa nu o vom putea muta in DAL (pentru ca foloseste chestii din alte parti si se creeaza dependinte circulare, which is bad :-) )